vars/default_vars.yml

```diff
@@ -391,3 +391,5 @@ is_rpi: False
 is_redhat: False
 is_fedora: False
 is_centos: False
+
+#rtc_id: ds3231
```

Comment:
Has the desired effect as I don't have a rtc attached
> TASK [1-prep : include_tasks] **************************************************
included: /opt/iiab/iiab/roles/1-prep/tasks/raspberry_pi.yml for 127.0.0.1
TASK [1-prep : Add a udev rule to transfer hwclock to system clock at dev creation] ***
skipping: [127.0.0.1]
TASK [1-prep : Check for needing to enable i2c rtc device in config.txt] *******
skipping: [127.0.0.1]
TASK [1-prep : Add a udev rule to transfer hwclock to system clock at dev creation] ***
skipping: [127.0.0.1]